ddtrace/monkey.py

```diff
@@ -44,8 +44,10 @@
     'httplib': False,
     'vertica': True,
 
+    # Even though instrumentation for web frameworks can be installed manually,
+    # instrumenting should be done idempotently
+    "django": True,
```

Comment:
```suggestion
    'django': True,
```

Reply:
hahah do I be consistent with the integrations above, or below!?!?

Reply:
above... because those are the correct ones.

I figure you are going to edit each of these lines to be `True` at some point, so get them migrated in the right direction.

Reply:
:)